### PR TITLE
[Feature] Upgrade HuMMan and SMC Reader

### PR DESCRIPTION
**Pull request by caizhongang**
- [x] Use derived keypoints from SMPL annotations for HuMMan
- [x] Bounding box bug fix 
- [x] (Width, Height) fix in smc_reader
- [x] Extrinsic fix in smc_reader
- [x] Transform SMPL to device coordinate system
- [x] Temporary support for testing on HuMMan 
- [x] Unit tests

### CODE REVIEW

File: mmhuman3d/data/data_converters/gta_human.py [hunk 1]
```diff
@@ -29,30 +30,33 @@ class GTAHumanConverter(BaseConverter):
...
-        dict(
-            type='PerspectiveCameras',
-            convention='opencv',
-            in_ndc=False,
-            focal_length=focal_length,
-            image_size=image_size,
-            principal_point=camera_center))
+    def __init__(self, *args, **kwargs):
+        super(GTAHumanConverter, self).__init__(*args, **kwargs)
+
```
Comment (kimren227) [inline mmhuman3d/data/data_converters/gta_human.py:35]: Do we need to specify where this name comes from as a comment?

File: mmhuman3d/data/data_structures/smc_reader.py [hunk 2]
```diff
@@ -215,7 +217,7 @@ def get_iphone_intrinsics(self, iphone_id=0, frame_id=0, vertical=True):
         if vertical:
             fx, fy = intrinsics[0, 0], intrinsics[1, 1]
             cx, cy = intrinsics[0, 2], intrinsics[1, 2]
-            H, W = self.get_iphone_color_resolution(vertical=False)
+            W, H = self.get_iphone_color_resolution(vertical=False)
```
Comment (kimren227) [inline mmhuman3d/data/data_structures/smc_reader.py:220]: nice catch!

**Review by kimren227 — APPROVED** [review]
Looks nice, let's wait for the test to finish before merging.

File: mmhuman3d/data/data_structures/smc_reader.py (hunk 2 above)
Comment (caizhongang) [reply mmhuman3d/data/data_structures/smc_reader.py:220]: thanks.

File: mmhuman3d/data/data_converters/gta_human.py (hunk 1 above)
Comment (caizhongang) [reply mmhuman3d/data/data_converters/gta_human.py:35]: Fixed in https://github.com/open-mmlab/mmhuman3d/pull/87/commits/6bcb5fa26c02b5fed031865afa9c6929106d6ac7.

**Pull request merged**